html/browsers/browsing-the-web/history-traversal/api-availability.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<title>API availability following history traversal</title>
+<meta charset=utf-8>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<p>Test requires popup blocker disabled</p>
+<div id=log></div>
+<script>
+var t = async_test();
+var hasNavigated = false;
+var child;
+t.step(function() {
+  child = window.open("resources/api-availability-1.html");
+  add_result_callback(function() {
```

Comment:
How is this different from t.add_cleanup?

Reply:
testharness.js invokes functions registered via `add_result_callback` following every sub-test (instead of following a specific sub-test), and errors thrown break the harness by precluding additional sub-tests and causing a harness-level timeout.

In other words, there's no good reason to use it here. Updated to `t.add_cleanup`